cmd/trayscale/app.go

```diff
@@ -223,6 +223,7 @@ func (a *App) update(s tsutil.Status) {
 		a.notify(online) // TODO: Notify on startup if not connected?
 		a.tray.SetOnlineStatus(online)
 	}
+	a.tray.Update(s)
```

Comment:
This is a pain point for me. The original path only sets the online status if the connection status has changed.

I'm not sure if this should be the case for all possible status updates. 

Either way, most likely this needs to be modified to just send the `tsutil.Status` to tray for it to handle updates.

Reply:
You could combine `SetOnlineStatus()` and `Update()` and pass `tsutil.Status` along with `a.online` to `Update()` to allow it to handle all system tray updates internally.

Reply:
Combined into `Update`. I'm not passing `a.online` along as it doesn't make sense to me if we have the whole `Status`.

Now we have to consider how to handle these updates not to waste CPU cycles. It will be updating with the rate of polling (5 seconds AFAICS). It's not ideal and the more items we use, the worse it will get.

I'm not yet sure what will be more expensive (provided we don't change polling rate or switch to the bus watcher #62) - holding last state in tray management and diffing on each update or sending updates to `systray` and hoping it figures refreshes out better than we do.

Reply:
The main reason for passing `a.online` through, though it would have to be done _before_ the previous `if`, was so that the tray updates could internally check if the status has changed like the `if` right above the call to `Update()` does. That way it doesn't have to change the text every time. It's got one extra `if` than separating the calls, but it lets the update be entirely internal and thus reduces the coupling. In fact, instead of passing `a.online` directly, you could pass `a.online != online` and call the argument `statusChanged` or something.

Reply:
OK, I see your point. I added the previous state as an argument in [a3363b6](https://github.com/DeedleFake/trayscale/pull/72/commits/a3363b6c65706c9e5972401d90d46896746c4c88).